src/test/isolation2/sql/udf_exception_blocks_panic_scenarios.sql

```diff
@@ -96,81 +72,82 @@ END; /* in func */
 $$
 LANGUAGE plpgsql;
 SELECT role, preferred_role, content, mode, status FROM gp_segment_configuration;
-SET debug_dtm_action_segment=0;
-SET debug_dtm_action_target=protocol;
-SET debug_dtm_action_protocol=subtransaction_begin;
-SET debug_dtm_action=panic_begin_command;
-SET debug_dtm_action_nestinglevel=0;
+--
+--
+SELECT gp_inject_fault('exec_mpp_dtx_protocol_command_start','panic','subtransaction_begin','','',1,-1,0,dbid)
```

Comment:
Please add white spaces after commas here and elsewhere in the delta to improve readability, like below:
```suggestion
SELECT gp_inject_fault('exec_mpp_dtx_protocol_command_start', 'panic', 'subtransaction_begin', '', '', 1, -1, 0, dbid)
```